src/validateEmail.test.js

```diff
@@ -8,11 +8,46 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('annA12.an$d@gmail.com')).toBe('boolean');
+  });
+
+  it(`should return 'true' for the valid email an12.a@gmail.com`, () => {
+    expect(validateEmail('an12.a@gmail.com')).toBeTruthy();
+  });
+
+  it(`should return 'true' for the valid email
+  with digits an12345sa@gmail.com`, () => {
+    expect(validateEmail('an12345sa.a@gmail.com')).toBeTruthy();
+  });
+
+  it(`should return 'false' for the email
+  with double dots an..45sa@gmail.com`, () => {
+    expect(validateEmail('an..45sa@gmail.com')).toBeFalsy();
```

Comment:
Maybe it means such double dot : ?

Reply:
in this case: it is a special symbol for me